gitbook-docs/data_model.md

```diff
@@ -171,9 +171,10 @@ A delta message can be recognised from the other types by the topmost level havi
 `updates` is the only required property.
 If `context` is missing it is assumed that the data is related to the `self` context.
 
-Context is a path from the root of the full tree. In this case 'vessels.urn:mrn:imo:mmsi:234567890'. All subsequent data is relative to
-that location. The context could be much more specific, e.g. 'vessels.urn:mrn:imo:mmsi:234567890.navigation', whatever is the common root
-of the updated data.
+Context is a path from the root of the full tree to the _container object_, which is always a vessel with the current Signal K version.
```

Comment:
This is true for current implementations, but the current message format allows for updates to other contexts such as `resources` e.g. to add/remove waypoints. Line 172 also describes the `self` context and if we continue to allow that (but see #282) we should allow that to be explicitly set as well.